examples/api/lib/material/floating_action_button/floating_action_button.2.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for FloatingActionButton
+
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MyApp());
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  static const String _title = 'FloatingActionButton Sample';
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: _title,
+      theme: ThemeData(colorSchemeSeed: const Color(0xff6750a4), useMaterial3: true),
```

Comment:
I'll see if I can get the seed color for the spec for we match

Reply:
That would be cool, I just matched card example seed color.

Reply:
good guess ;)